blockstore/src/main/java/brooklyn/location/blockstore/NewVolumeCustomizer.java

```diff
@@ -18,7 +22,41 @@
 
 import java.util.List;
 
-public abstract class NewVolumeCustomizer extends BasicJcloudsLocationCustomizer {
+/**
+ * Creates a location customizer that:
+ * <ul>
+ * <li>configures the EC2 availability zone</li>
+ * <li>attaches the specified (existing) volume to the newly-provisioned EC2 instance</li>
+ * <li>mounts the filesystem under the requested path</li>
+ * </ul>
+ *
+ * Can be used for attaching additional disk on provisioning time for AWS.
+ * Below is shown an example:
+ *
+ * <pre>
+ *   provisioning.properties:
+ *     customizers:
+ *     - $brooklyn:object:
+ *         type: io.brooklyn.blockstore.brooklyn-blockstore:brooklyn.location.blockstore.ec2.Ec2NewVolumeCustomizer
```

Comment:
The example here is with `Ec2NewVolumeCustomizer` which is not appropriate for other clouds. If it should be `NewVolumeCustomizer`, better correct it here.